packages/react/App.tsx

```diff
@@ -0,0 +1,223 @@
+import React from 'react';
+import { CdsAlert, CdsAlertActions, CdsAlertGroup } from './src/alert';
+import { CdsButton, CdsButtonType } from './src/button';
+import { CdsBadge } from './src/badge';
+import { CdsIcon } from './src/icon';
+import { CdsModal, CdsModalActions, CdsModalContent, CdsModalHeader } from './src/modal';
+import { CdsTag } from './src/tag';
+import { ClarityIcons, userIcon, timesIcon } from '@clr/core/icon';
+import '@clr/core/icon/register';
+
+ClarityIcons.addIcons(userIcon, timesIcon);
+
+interface AppProps {}
+interface AppState {
+  modalOpen: boolean;
+}
+
+export default class App extends React.Component<AppProps, AppState> {
+  buttonRef: any;
+
+  constructor(props) {
+    super(props);
+    this.state = { modalOpen: false };
+    this.buttonRef = React.createRef<CdsButtonType>();
+  }
+
+  componentDidMount() {
+    this.buttonRef.current.nativeElement.then(element => {
+      element.focus();
+    });
+  }
+
+  handleOnCloseChange(e: any) {
+    console.log(e);
+  }
+
+  handleCloseModalClick() {
+    this.setState({ modalOpen: false });
+  }
+
+  handleOpenModalClick() {
+    this.setState({ modalOpen: true });
+  }
+
+  handleSetModalState(state: boolean) {
+    this.setState({ modalOpen: state });
+  }
+
+  render() {
+    const isModalOpen = this.state.modalOpen;
+    return (
+      <div>
+        <h1>Rendered by React!</h1>
+        <CdsButton ref={this.buttonRef} onClick={this.handleOpenModalClick.bind(this, true)}>
+          Open Modal
+        </CdsButton>
+        {isModalOpen ? (
+          <CdsModal onCloseChange={this.handleCloseModalClick.bind(this, false)}>
+            <CdsModalHeader>
+              <h3 cds-text="title">My Modal</h3>
+            </CdsModalHeader>
+            <CdsModalContent>
+              <div cds-layout="vertical gap:md p-y:xs">
+                <p cds-text="body">Lorem Ipsum</p>
+              </div>
+            </CdsModalContent>
+            <CdsModalActions>
+              <div cds-layout="horizontal gap:sm align:right">
+                <CdsButton onClick={this.handleCloseModalClick.bind(this, false)} action="outline">
+                  Cancel
+                </CdsButton>
+                <CdsButton onClick={this.handleCloseModalClick.bind(this, false)}>Ok</CdsButton>
+              </div>
+            </CdsModalActions>
+          </CdsModal>
+        ) : (
+          <br></br>
```

Comment:
Why is this `<br></br>` and not `<br/>` like below on line 213?

Reply:
No good reason. Will clean this up.